### PR TITLE
Fix #583: Refactored Overview to Info

### LINKED ISSUE #583: Overview should be updated to Info
As per new mocks, Topic Overview Tab has been updated to Topic Info Tab,

Screenshot 2020-01-03 at 12 11 29 PM

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Overview` with `Info`.

### PR DESCRIPTION
**Pull request by Abhishek0706**
Mock Link:
https://user-images.githubusercontent.com/9396084/71710439-3fa89500-2e22-11ea-83c9-a5564e000459.png

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +165,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: Please revert all the changes from this file, it might happen that it is not possible.

**Review by rt4914 — APPROVED** [review]
LGTM. (Looks good to me.)

**Review by rt4914 — CHANGES_REQUESTED** [review]
Actually, blocking this PR for now, please make following changes:

1. Change the PR title to: `Fix #583: Refactored Overview to Info`
2. Also, I think you are unable to assign anyone in `Assignees` and `Reviewers` section present on the right side of this pull request. In that case you make a top level comment like this: `@rt4914 PTAL` meaning "Please take a look".
3. As mentioned in earlier comment, revert the changes from `Project.xml` file.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (Abhishek0706) [reply .idea/codeStyles/Project.xml]: Done!

**Review by rt4914 — APPROVED** [review]
LGTM

**Comment by rt4914** [comment]
@Abhishek0706 This PR looks good. Congratulations on your first contribution. Now you can merge this PR using `Squash and merge` option visible on this PR and delete the branch after that.

**Comment by rt4914** [comment]
> @Abhishek0706 This PR looks good. Congratulations on your first contribution. Now you can merge this PR using `Squash and merge` option visible on this PR and delete the branch after that.

I will need to merge this as you are not currently a contributor to team yet.

**Pull request merged**